x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,116 @@ export const PolicyList = React.memo(() => {
+            </>
+          );
         },
         truncateText: true,
       },
       {
-        field: 'revision',
-        name: i18n.translate('xpack.siem.endpoint.policyList.revisionField', {
-          defaultMessage: 'Revision',
+        field: 'created_by',
+        name: i18n.translate('xpack.siem.endpoint.policyList.createdBy', {
+          defaultMessage: 'Created By',
         }),
-        dataType: 'number',
+        truncateText: true,
       },
       {
-        field: 'package',
-        name: i18n.translate('xpack.siem.endpoint.policyList.versionField', {
-          defaultMessage: 'Version',
+        field: 'created_at',
+        name: i18n.translate('xpack.siem.endpoint.policyList.createdAt', {
+          defaultMessage: 'Created Date',
         }),
-        render(pkg) {
-          return `${pkg.title}  v${pkg.version}`;
+        render(createdAt: string) {
+          return <FormattedDateAndTime date={new Date(createdAt)} />;
         },
       },
       {
-        field: 'description',
-        name: i18n.translate('xpack.siem.endpoint.policyList.descriptionField', {
-          defaultMessage: 'Description',
+        field: 'updated_by',
+        name: i18n.translate('xpack.siem.endpoint.policyList.updatedBy', {
+          defaultMessage: 'Last Updated By',
         }),
         truncateText: true,
       },
       {
-        field: 'config_id',
-        name: i18n.translate('xpack.siem.endpoint.policyList.agentConfigField', {
-          defaultMessage: 'Agent Configuration',
+        field: 'updated_at',
+        name: i18n.translate('xpack.siem.endpoint.policyList.updatedAt', {
+          defaultMessage: 'Last Updated',
         }),
-        render(version: string) {
-          return (
-            <LinkToApp
-              data-test-subj="agentConfigLink"
-              appId="ingestManager"
-              appPath={`#/configs/${version}`}
-              href={`${services.application.getUrlForApp('ingestManager')}#/configs/${version}`}
-            >
-              {version}
-            </LinkToApp>
-          );
+        render(updatedAt: string) {
+          return <FormattedDateAndTime date={new Date(updatedAt)} />;
         },
       },
+      {
+        field: 'package',
+        name: i18n.translate('xpack.siem.endpoint.policyList.versionFieldLabel', {
+          defaultMessage: 'Version',
+        }),
+        render(pkg) {
+          return i18n.translate('xpack.siem.endpoint.policyList.versionField', {
+            defaultMessage: '{title} v{version}',
+            values: {
+              title: pkg.title,
+              version: pkg.version,
+            },
+          });
+        },
+      },
+      {
+        field: '',
+        name: 'Actions',
+        actions: [
+          {
+            // eslint-disable-next-line react/display-name
+            render: (item: Immutable<PolicyData>) => {
+              return (
+                <EuiContextMenuItem icon="link" key="datasourceEdit">
```

Comment:
do you need to use `<EuiPopover>` to wrap this which handles ensuring the EuiContext menu is closed if user clicks somewhere else?

Also, you are returning `<EuiContextMenuItem>` but its not wrapped in a `<EuiContextMenu>`?

Reply:
There is no `EuiPopover` just yet.  There will be after there are more than 3 actions.  Otherwise, I need to create a custom component just like in the ingest solution.

Good call on `EuiContextMenuItem`.  It's left over from some experimentation.  Using just the Link and Icon now, will push that up without the menu item

Reply:
@kevinlog 

#### Re: `EuiPopover`
Ok. I think perhaps we are thinking differently about the actions. I was looking at our custom implementation as a **replacement** of the auto-layout switch that `EuiBasicTable` uses. So we would define our own Component (essentially a context menu wrapper around the popover) that will handle all menu options. The table should ever have at most 1 action === our custom Component (at least that is the approach I took in ingest).

FYI: you can in fact have a `<Popover>` on the page even if the menu is not shown - that component is smart enought to only intercept clicks outside of the menu when its "clicked" on. Here is the implementation from:

https://github.com/elastic/kibana/blob/8c5071939b8bf3338128539b23f7b9db43c8798c/x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/components/table_row_actions.tsx#L19-L35